android/assets/jsons/translations/template.properties

```diff
@@ -1322,8 +1322,14 @@ ConditionalsPlacement =
 # As this is still under development, conditionals will be added al the time. As a result,
 # any translations added for this string will be removed immediately in the next version when more
 # conditionals are added. As we don't want to make you retranslate this same line over and over,
-# it's removed for now, but it will return once all planned conditionals have been added.
+# it's removed for now, but it will return once all planned conditionals have been added.  
+
 
+# <if this city has at least [amount] specialists> <when at war> <when not at war> <while the empire is happy> = 
+if this city has at least [amount] specialists = se la città possiede almeno [amount] specialisti
```

Comment:
Looks like testing lines meant for Italian.properties, but this is template.properties